easybuild/easyblocks/g/gromacs.py

```diff
@@ -102,6 +105,14 @@ def get_gromacs_arch(self):
             # According to [2] the performance difference between SSE2 and SSE4.1 is minor on x86
             # and SSE4.1 is not supported by AMD Magny-Cours[1].
             res = 'SSE2'
+        elif optarch == OPTARCH_GENERIC:
+            cpu_arch = get_cpu_architecture()
+            if cpu_arch == X86_64:
+                res = 'SSE2'
+            else:
+                res = 'None'
+        elif optarch:
+            self.log.info("OPTARCH set to %s but not used. Compiling GROMACS for the current host architecture")
```

Comment:
@victorusu Maybe using `print_warning` makes more sense here, to make it pop up in the output of `eb` (rather than only in the log file)?

Also, the `%s` isn't being templated... And I would use `--optarch configuration option` rather than `OPTARCH`